### PR TITLE
Update `readme.md` including link to jpegoptim's requirements

### PR DESCRIPTION
**Pull request by philipbordallo**
imagemin/imagemin-jpegoptim#8

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # jpegoptim-bin [![Build Status](http://img.shields.io/travis/imagemin/jpegoptim-bin.svg?style=flat)](http://travis-ci.org/imagemin/jpegoptim-bin)
 
-> jpegoptim is a utility for optimizing JPEG files that provides lossless optimization (based on optimizing the Huffman tables) and "lossy" optimization based on setting a maximum quality factor
+> [jpegoptim](https://github.com/tjko/jpegoptim) is a utility for optimizing JPEG files that provides lossless optimization (based on optimizing the Huffman tables) and "lossy" optimization based on setting a maximum quality factor
 
 
 ## Install
 
-```
+```sh
```
Comment (shinnn) [inline readme.md]: Don't add `sh` highlight helper. Github's shell syntax highlight is not sophisticated.

For example:

```sh
npm i read-json-sync
```

`read` is unexpectedly highlighted.

File: readme.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 $ npm install --save jpegoptim-bin
 ```
 
+Make sure you have the correct version of libjpeg. See [jpegoptim's README](https://github.com/tjko/jpegoptim#readme) for more info.
```
Comment (shinnn) [inline readme.md]: `for more info` → `for more information`

**Comment by shinnn** [comment]
Can you squash those two commits into a single commit? `push --force` to `philipbordallo:master` branch will update the pull request.

**Comment by philipbordallo** [comment]
@shinnn Yep! Apologies for the mixups.

**Pull request merged**